### DISCUSSION TITLE
lightzoneproject.org seems to be down

### DISCUSSION THREAD

**Discussion by jbleuel** [Aries85/LightZone#302] [General]
I saw lightzoneproject.org is down since a while and only the hosting provider screen shows up (Bluehost: Affordable, Reliable
Web Hosting Solutions).

Is this known and/or temporarily and/or is there a plan for a new home?

Thanks a lot for this great software that I use since years !!!

All the best,
Jens

**Comment by sfink16** [comment]
Yes, the original LZ forum maintainer no longer responded to request and apparently abandoned the website.  No one knows why, but hopefully it was not health or a catastrophic situation.  The best place to find information either in this discussion section, which obviously you found, or the [Pixls.us](https://discuss.pixls.us/c/software/lightzone/40) website found in the [Wiki](https://github.com/ktgw0316/LightZone/wiki) section of this site.
